sdk/storage/azure-storage-common/CHANGELOG.md

```diff
@@ -4,10 +4,14 @@
 
 ### Features Added
 
+- Used new xml library on Windows, dropped dependency for libxml2.
```

Comment:
How is this a new feature?

Reply:
Maybe. But we've shipped the release, it's too late.

Reply:
No worries, for next time :)